eg/Classes/ConfigPanel.py

```diff
@@ -51,7 +51,9 @@ def __init__(
         self.isDirty = False
         self.resultCode = None
         self.buttonsEnabled = True
-        self.dialog.buttonRow.applyButton.Enable(False)
+        self.dialog.buttonRow.applyButton.Enable(
```

Comment:
While i was testing the PR i came across plugins that don't have settings to change but still show a config dialog (File Operations for example). In that case it would be nice to have the Ok buton enabled. I found the following solution:
first you need to import
```python
from inspect import currentframe, getargvalues, getouterframes
```
and then change line 54-59 to
```python
        outer = getouterframes(currentframe())
        has_args = len(getargvalues(outer[1][0]).args) > 1
        self.dialog.buttonRow.applyButton.Enable(
            has_args and self.dialog.treeItem.isFirstConfigure
        )
        self.dialog.buttonRow.okButton.Enable(
            not has_args or
            not (has_args and self.dialog.treeItem.isFirstConfigure)
        )
```
What do you think about adding this?

Reply:
do it in a separate PR. if for some reason there is an issue with it then it can be easily reverted.. I do like that idea tho.

which will bring me to point out an old PR. #141